### ISSUE TITLE
Make a icon

### ISSUE THREAD

**Issue by genkimarshall** [genkimarshall/wwwild#4]
Totally open to ideas :P

**Comment by kylecoburn** [comment]
Some random ideas:

* WWW (cons: can look like M's on the bottom)

![logo-1-small](https://cloud.githubusercontent.com/assets/6035825/7217920/5f96a9e2-e601-11e4-88bd-1b223279d55a.png)

* A stylized wildflower

**Comment by genkimarshall** [comment]
+1 the idea of making the icon around the 3 w's.
Of course, the design of wwwild's interface is currently very not-done, but I do feel like a black background doesn't match it at least for the time being..
Maybe there's a way to combine the idea of the first two?
Thanks a ton for kicking this off, I'll think about it more :D

**Comment by kylecoburn** [comment]
Agreed, fonts/colors would want to change. (I just tend to start with a black background for prototyping icons to make details easier to distinguish.)

And it _could_ look like a flower depending on how it is done. Sounds hard to pull off, but would be neat!

**Comment by genkimarshall** [comment]
Yeah for sure, makes sense!
Ha yeah it's kinda hard to imagine but maybe..
